src/app/pages/home/home.component.html

```diff
@@ -66,181 +68,123 @@ <h2 class="mb-3 px-5 home-text header-text">What is a virtrolio?</h2>
-          </li>
-        </ul>
-        <h5>Formatting</h5>
-        <p>
-          Virtrolio supports a modified version of <em>Markdown</em> to format text. Learn how to type in markdown <a
-          href="https://www.markdownguide.org/basic-syntax/" target="_blank">here</a>. ️
-        </p>
+<div class="full-width demo-section" id="signing-demo">
+  <div data-aos="fade-right" data-aos-duration="2000" data-aos-once="true"
+       class="full-width demo-image d-none d-md-block signing-demo-image">
+    <div class="vertical-center mx-5 px-5 signing-demo-text-container">
+      <h2 class="demo-title">Give your messages</h2>
+      <h2 class="demo-title">your <i>signature</i> touch</h2>
+    </div>
+  </div>
+  <div class="full-width demo-image d-block d-md-none signing-demo-image">
+    <div class="vertical-center mx-5 px-5 signing-demo-text-container">
+      <h2 class="demo-title">Give your messages</h2>
+      <h2 class="demo-title">your <i>signature</i> touch</h2>
+    </div>
+  </div>
+</div>
 
-        <h5>Emojis</h5>
-        <p>We also use an emoji library, so you can press (Windows + Period) on a Windows computer or (Command + Control
-          + Space) on a Mac to bring up the emoji
-          keyboard. Note: not all emojis may be supported; check <a
-            href="https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md" target="_blank">this link of
-            supported emojis</a> to make sure! ❤</p>
-      </div>
-      <div class="modal-footer">
-        <button class="btn btn-secondary" data-dismiss="modal" type="button">Close</button>
+<div class="py-1 vertical-center body-container">
+  <div class="row d-flex justify-content-center">
+    <div class="col-12 col-sm-12 col-md-12 col-lg-8 col-xl-8" data-aos="zoom-in" data-aos-duration="1000"
+         data-aos-once="true">
+      <div class="px-5" id="login-virtrolio-button-bottom">
+        <!-- Call to sign in and Google sign in button; displays if logged out -->
+        <div *ngIf="!authService.isLoggedIn() else MyVirtrolio">
+          <div class="justify-content-center text-center pt-1">
+            <h3 class="mb-1 px-5 home-text ready-text">Ready to get signing?</h3>
+            <p class="lead pb-1">Sign in and get your virtrolio!</p>
+          </div>
+          <div class="row d-flex justify-content-center mt-4">
+            <button (click)="authService.login('/my-virtrolio')" class="btn btn-outline-dark google-sign-in-btn"
+                    type="button">
+              <img alt="Google Logo" height="23" src="../../../assets/images/icons/google.svg" width="23">
+              <span>Sign in with Google</span>
+            </button>
+          </div>
+        </div>
+
+        <!-- Button to replace 'Sign in with Google' when you're signed in -->
+        <ng-template #MyVirtrolio>
+          <div class="justify-content-center text-center pt-1">
+            <p class="lead pb-1">You're already signed in! Click the button to go to your virtrolio.</p>
```

Comment:
Change this to be slightly different from the 'go to my virtrolio' subtitle text that's above the first go to my virtrolio button—this looks like a misplaced copy paste right now.